### ISSUE TITLE
Adding a "Stripe-Account" header

### ISSUE THREAD

**Issue by emhoracek** [dmjio/stripe#67]
First off, thanks for maintaining this great library!

So, I have an app with a Stripe account that needs to make charges for and charge fees to other accounts. So, we can use [Stripe Connect](https://stripe.com/docs/connect) for that! And just using API keys  from the other accounts along with the ApplicationFee param works great. 

However, Stripe _prefers_ that apps use [this form of authentication instead](https://stripe.com/docs/connect/authentication#authentication-via-the-stripe-account-header), a header with a Stripe account id. 

I can't figure out if stripe-haskell doesn't have this capability or if I'm just missing it? 

Appreciate any help!

**Comment by dmjio** [comment]
Hi @emhoracek, and thank you :) 

Unfortunately, this library does not (yet?) support Stripe Connect. It was actually added to the old `stripe` library by @LukeHoersten. If you ask nicely maybe he'll make a PR into this project :) 

I'm not too familiar with your specific use case, but if your use case is simply to add a header to a request I can most likely create functionality to accommodate your use case. Let me know. I'm not as familiar with stripe connect, since at the time (2014), connect's docs were separate from the main stripe API docs.

**Comment by aviaviavi** [comment]
Hi @dmjio - Just checking, is Stripe Connect still unsupported by this library? I don't see much in this repo for it, but just wanted to make sure I'm not missing it somewhere before writing the bindings for that part of the API myself.

**Comment by dmjio** [comment]
@aviaviavi that's correct. PRs always welcome. Let me know if you want to collaborate on it.
